src/transformers/models/opt/modeling_opt.py

```diff
@@ -1173,7 +1158,36 @@ def forward(
             Labels for position (index) of the end of the labelled span for computing the token classification loss.
             Positions are clamped to the length of the sequence (`sequence_length`). Position outside of the sequence
             are not taken into account for computing the loss.
-        """
+
+        Returns:
+
+        Example:
+
+        ```python
+        >>> from transformers import GPT2Tokenizer, OPTForQuestionAnswering
+        >>> import torch
+
+        >>> torch.manual_seed(4)  # doctest: +IGNORE_RESULT
+        >>> tokenizer = GPT2Tokenizer.from_pretrained("facebook/opt-350m")
+
+        >>> # note: we are loading a OPTForQuestionAnswering from the hub here,
+        >>> # so the head will be randomly initialized, hence the predictions will be random
+        >>> model = OPTForQuestionAnswering.from_pretrained("facebook/opt-350m")
+
+        >>> question, text = "Who was Jim Henson?", "Jim Henson was a nice puppet"
```

Comment:
```suggestion
        >>> question = "Who was Jim Henson?"
        >>> text =  "Jim Henson was a nice puppet"
```
Would just change that for clarity 😉

Reply:
Well, it is taken from `PT_QUESTION_ANSWERING_SAMPLE` in `src/transformers/utils/doc.py`. I would like to keep as it is at this moment, and we can have a discussion to change to 2 lines for all occurrences later.

Reply:
Going to merge in 1h2m3s, if no further comment from Arthur :-)